control-plane/api/v1alpha1/mesh_types.go

```diff
@@ -72,7 +72,7 @@ type MeshTLSConfig struct {
 type MeshDirectionalTLSConfig struct {
 	// TLSMinVersion sets the default minimum TLS version supported.
 	// One of `TLS_AUTO`, `TLSv1_0`, `TLSv1_1`, `TLSv1_2`, or `TLSv1_3`.
-	// If unspecified, Envoy v1.22.0 and newer will default to TLS 1.2 as a min version,
+	// If unspecified, Envoy v1.22.2 and newer will default to TLS 1.2 as a min version,
```

Comment:
As Nitya mentioned, we dont need to update this comment as it is valid. This inadvertently makes it slightly less valid as this behavior will also be observed in previous versions. Can we return this to it's previous value `v1.22.0` and run `make ctrl-manifests` as @ndhanushkodi recommended. We can merge it after!

Reply:
Haha, got it. I was just searching and replacing. That makes sense. Thanks for the feedback!